creator-node/scripts/run-tests.sh

```diff
@@ -11,7 +11,6 @@ do
   esac
 done
 
-IPFS_CONTAINER=cn-test-ipfs-node
```

Comment:
this is what i was referring to with the CI change with the test runner in CI still bringing up the IPFS pod. we can make it in this PR or follow up https://github.com/AudiusProject/audius-protocol/blob/2a37bcff1bb1a82efdf187d1723b3457dc0dcb9b/.circleci/config.yml#L389

Reply:
made a note to do a follow up PR